### PR TITLE
Added emoji support for Trump tweets

### PR DESCRIPTION
**Pull request by Bezbakri**
Ignore my bumbling second commit
Also, if you don't have pilmoji, the library, installed, do
`pip install pilmoji`
otherwise, the code won't work

### CODE REVIEW

**Comment by dovedevic** [comment]
Will need to wait for FAN to be upgraded to `nextcord` as it is currently running on py36 and this package requires py38. When the `nextcord` update comes, I'll be upgrading to py310.

**Comment by Bezbakri** [comment]
Hello dove, the emoji thingy was removed, so no need for an update. Also, added the funny morbius filter.

**Comment by dovedevic** [comment]
Just use the apply mask method thats already present so we can do resizing and gif support :)

File: src/fullyautomatednutcracker/cogs/filters.py [hunk 1]
```diff
@@ -1001,5 +1001,20 @@ async def djtj(self, ctx, *, text = None):
...
+            return
+        img = await self.get_asset_from_user(ctx, image_link)
+        if not img:
+            return
+        img = img.convert("RGBA")
+        morb_img = img.resize((640, 614))
+        morb_mask = Image.open("content/filters/morbius.png").convert("RGBA")
+        morb_img.paste(morb_mask, (0,0), morb_mask)
+        
+        await self.save_img_and_send(arguments, ctx.author, ctx.channel, morb_img, file_name="morbius", things_to_close=(img, morb_img, morb_mask))
```
Comment (dovedevic) [inline src/fullyautomatednutcracker/cogs/filters.py]: Just replace all this to

```python
await self.apply_mask(ctx, "morbius.png")
```

**Review by dovedevic — CHANGES_REQUESTED** [review]

File: src/fullyautomatednutcracker/cogs/filters.py (hunk 1 above)
Comment (Bezbakri) [reply src/fullyautomatednutcracker/cogs/filters.py]: Done. Should be fine now.

**Review by dovedevic — APPROVED** [review]

**Pull request merged**